go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/briandowns/spinner v1.18.1
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/finleap-connect/monoskope v0.3.6-dev20
+	github.com/finleap-connect/monoskope v0.3.6-dev41
```

Comment:
Before merging I would like that we have none dev/rc versions here. So that we first release the monoskope version and then update the dependency here and create a release here. What do you think?

Reply:
I would second that, but we still have [getByUser](https://github.com/finleap-connect/monoskope/pull/127) and I would merge this one to branch out of main to write the client-side of getByUser